garrysmod/lua/includes/modules/utf8_case.lua

```diff
@@ -0,0 +1,15 @@
+
+local uc_lc = {}
+for lc, uc in pairs( lc_uc ) do uc_lc[ uc ] = lc end
```

Comment:
You could use [table.Flip](https://wiki.facepunch.com/gmod/table.Flip) here ![](https://cdn.discordapp.com/emojis/1073702338495270912.png?size=16)